compiler/damlc/tests/daml-test-files/InterfaceChoiceGuardFailed.daml

```diff
@@ -3,7 +3,7 @@
 
 module InterfaceChoiceGuardFailed where
 
--- @SINCE-LF-FEATURE DAML_INTERFACE
+-- @SINCE-LF-FEATURE DAML_INTERFACE_EXTENDED
```

Comment:
For tests that only make sense with `DAML_INTERFACE_EXTENDED`, I'd appreciate having an identical test but with the following header:

```
-- @SINCE-LF-FEATURE DAML_INTERFACE
-- @UNTIL-LF-FEATURE DAML_INTERFACE_EXTENDED
-- @ERROR Daml Interfaces with guards and requires are only available with --target=1.dev
```